DataRepo/templates/DataRepo/includes/animal_list_stats_table.html

```diff
@@ -0,0 +1,93 @@
+{% load customtags %}
+
+{% if out_df %}
+<div>
+    <table class="table table-sm table-hover table-bordered table-responsive-xl table-striped"
+        id="animal_list_stats"
+        data-toggle="table"
+        data-buttons-class="primary"
+        data-buttons-align="left"
+        data-filter-control="true"
+        data-search="true"
+        data-search-align="left"
+        data-show-search-clear-button="true"
+        data-show-multi-sort="true"
+        data-show-columns="true"
+        data-show-columns-toggle-all="true"
+        data-show-fullscreen="true"
+        data-show-export="true"
+        data-export-types="['csv', 'txt', 'excel']"
+        data-height="1200"
+        data-virtual-scroll="true"
+        data-pagination="true"
+        data-page-size="25"
+        data-page-list="[25, 50, 100, 200, 500, All]">
+        <thead>
+            <tr>
+                <th data-filter-control="input" data-sortable="true" data-sorter="htmlSorter" data-field="Animal">Animal</th>
+                <th data-filter-control="input" data-sortable="true" data-sorter="htmlSorter" data-field="Studies">Studies</th>
+                <th data-filter-control="select" data-sortable="true" data-field="Genotype">Genotype</th>
+                <th data-filter-control="input" data-field="Infusate">Infusate</th>
+                <th data-filter-control="input" data-sortable="true" data-sorter="htmlSorter" data-field="Compounds">Parent Compound(s) for Tracer(s)</th>
+                <th data-filter-control="input" data-sortable="true" data-sorter="htmlSorter" data-field="Tracers">Tracer(s)</th>
+                <th data-filter-control="input" data-sortable="true" data-field="Concentrations">Infusion Concentration(s) (mM)</th>
+                <th data-filter-control="select" data-sortable="true" data-field="Labeled-Elements">Labeled Element(s)</th>
+                <th data-filter-control="input" data-sortable="true" data-field="Infusion-Rate">Infusion Rate (ul/min/g)</th>
+                <th data-filter-control="input" data-sortable="true" data-sorter="htmlSorter" data-field="Treatment">Treatment</th>
+                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Body_Weight">Body Weight (g)</th>
+                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Age">Age (weeks)</th>
+                <th data-filter-control="select" data-sortable="true" data-visible="false" data-field="Sex">Sex</th>
+                <th data-filter-control="input" data-sortable="true" data-visible="false" data-field="Diet">Diet</th>
+                <th data-filter-control="input" data-sortable="true" data-field="Feeding-Status">Feeding Status</th>
+                <th data-filter-control="select" data-sortable="true" data-field="Sample-Owners">Sample Owners</th>
```

Comment:
NB. If sample owner == animal owner, then...
```suggestion
                <th data-filter-control="select" data-sortable="true" data-field="Sample-Owners">Owner(s)</th>
```